R/process_observers.R

```diff
@@ -280,14 +295,17 @@ confirm_observer_NA <- function(rocombos, mbbs_county, county_observer_table, su
 #' @importFrom utils write.csv
 update_mini_observer_table <- function() {
   # load the main observer conversion table
-  observer_table <- read.csv("inst/extdata/main_observer_conversion_table.csv", header = TRUE)
+  observer_table <-
+    read.csv("inst/extdata/main_observer_conversion_table.csv", header = TRUE)
   # load the mini observer conversion table (for obs1,obs2,obs3)
-  mini_observer_table <- read.csv("inst/extdata/mini_observer_conversion_table.csv", header = TRUE)
+  mini_observer_table <-
+    read.csv("inst/extdata/mini_observer_conversion_table.csv", header = TRUE)
```

Comment:
We should consider using system.file instead of explicitly giving the path. This can be addressed in separate pr.